agent/agent.go

```diff
@@ -87,6 +88,9 @@ func New(options Options) io.Closer {
 	if options.TempDir == "" {
 		options.TempDir = os.TempDir()
 	}
+	if options.LogDir == "" {
+		options.LogDir = options.TempDir
```

Comment:
nit: I would leave here a log message in favor of future debugging and support.

Reply:
A message about what? That we defaulted to options.TempDir? Sure I can add that. 👍🏻